trident/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora
 
-ENV TRIDENT_VERSION='21.10.1'
+ENV TRIDENT_VERSION='22.04.0'
```

Comment:
What is this Dockerfile for, from what I can see we are using upstream images?

Reply:
This is actually your idea, commit: 8d27b54ebc41bf253bd2368cee03dbdec5e2d73c, to automate running tridentctl inside a container to generate manifests.

Reply:
I forgot :D